app/config/sculpin_kernel.yml

```diff
@@ -6,11 +6,11 @@ sculpin_content_types:
     guides:
         permalink: documentation/guides/:slug_title.html
 
-    libraries:
-        permalink: documentation/libraries/:slug_title.html
+    # libraries:
+    #     permalink: documentation/libraries/:slug_title.html
```

Comment:
do we have a jira ticket to keep track of this hack? #17 is adding to the libraries and it looks to me like that will get lost with this.